tests/providers/docker/operators/test_docker_swarm.py

```diff
@@ -345,3 +351,46 @@ def test_service_args_list(self, types_mock, docker_api_client_patcher):
             configs=None,
             secrets=None,
         )
+
+    @mock.patch("airflow.providers.docker.operators.docker_swarm.types")
+    def test_logging_driver(self, types_mock, docker_api_client_patcher):
+        mock_obj = mock.Mock()
+
+        client_mock = mock.Mock(spec=APIClient)
+        client_mock.create_service.return_value = {"ID": "some_id"}
+        client_mock.images.return_value = []
+        client_mock.pull.return_value = [b'{"status":"pull log"}']
+        client_mock.tasks.return_value = [{"Status": {"State": "complete"}}]
+        types_mock.TaskTemplate.return_value = mock_obj
+        types_mock.ContainerSpec.return_value = mock_obj
+        types_mock.RestartPolicy.return_value = mock_obj
+        types_mock.Resources.return_value = mock_obj
+
+        docker_api_client_patcher.return_value = client_mock
+
+        operator = DockerSwarmOperator(
+            image="", logging_driver="json-file", task_id="unittest", enable_logging=False
+        )
+        operator.execute(None)
```

Comment:
we probably would want to assert that the logging drive is "json-file" here
```suggestion
        assert operator.logging_driver == "json-file"
```

Reply:
okay, I have updated the test and confirmed that it passed.